apf-ingester/apf_Ingester.py

```diff
@@ -0,0 +1,264 @@
+import re
+import os
+import json
+import shutil
+import tempfile
+import pathlib
+from vistautils.parameters import Parameters
+from vistautils.parameters_only_entrypoint import parameters_only_entry_point
+from typing import Dict, List, Optional
+
+
+def cleanup_source_document(sgm_path: str) -> None:

```

Comment:
In-place changes should be avoided - prefer to write to a new location and return the path to it, or if possible just work with strings in-memory.

Reply:
Function removed (unecessary, as mentioned above)